src/components/@shared/FormFields/FilesInput/Info.tsx

```diff
@@ -19,7 +19,7 @@ export default function FileInfo({
   // Prevent accidential publishing of error pages (e.g. 404) for
   // popular file hosting services by warning about it.
   // See https://github.com/oceanprotocol/market/issues/1246
-  const shouldWarnAboutFile = file.valid && contentTypeCleaned === 'html'
```

Comment:
if none of this is used, all the comments and commented out code should be removed